test-requirements.txt

```diff
@@ -0,0 +1 @@
+odoo-addon-sale_commercial_partner @ git+https://github.com/OCA/sale-workflow.git@refs/pull/3415/head#subdirectory=sale_commercial_partner
```

Comment:
afaics was closed and replaced by https://github.com/OCA/sale-workflow/pull/3504 which is already merged - this can be removed